src/services/user.ts

```diff
@@ -0,0 +1,7 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/User';
+
+export function getUserById(userId: number):User | null {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
```

Comment:
null is redundant here
```suggestion
export function getUserById(userId: number):User {
  return usersFromServer.find(user => user.id === userId);
}
```